src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -35,6 +35,7 @@ import {colors} from '../../utils/colors';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
 import {getChatName} from '../../utils/string/StringUtils';
 import {isContainUrl} from '../../utils/Utils';
+import useProfileHook from '../../hooks/core/profile/useProfileHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useProfileHook` is a good addition as it allows the retrieval of the `anonProfileId` which is used later in the code. This helps to identify anonymous users and adjust the display accordingly.
```diff
+ import useProfileHook from '../../hooks/core/profile/useProfileHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->